docs/web/sdk-components-styling.mdx

```diff
@@ -116,20 +126,189 @@ export default function ComponentWithIcon(props) {
                     secondary={"cloud-download icon"}
                 />
                 <MenuItemSecondaryAction>
-                    // highlight-start
                     <IconButton title="IconExample">
-                        <DynamicIcon src="cloud-download" size="large" />
+                        // highlight-next-line
+                        <DynamicIcon src="cloud-download" fontSize="large" />
```

Comment:
In practice, you would never do this. If you know the ID upfront, then by definition it's a static icon and you should use `<CloudDownload/>` instead. `DynamicIcon` will incur the cost of an unnecessary extra request. Even though this is just sample code, it might be clearer if DynamicIcon was used as intended. Maybe add an "iconId" prop or something to your layout component and pass that to `src`? Then it will be clearer that DynamicIcon is to be used when the icon is not statically known at compile time.

Reply:
Ahh yes, good point. I've refactored the `DynamicIcon` examples to always take the iconId from a prop.